README.md

```diff
@@ -134,30 +134,41 @@ The goal of this library is to provide an easy to use interface of Bolt Cloud AP
     
     ```python
     from boltiot import Email
-    mailgun_api_key = 'This is the private API key which you can find on your Mailgun Dashboard' 
-    sandbox_url= 'You can find this on your Mailgun Dashboard' 
+    mailgun_api_key = 'This is the private API key which you can find on your Mailgun Dashboard'
+    sandbox_url= 'You can find this on your Mailgun Dashboard'
     sender_email = 'This would be test@your SANDBOX_URL'
     recipient_email = 'Enter your Email ID Here'
     mailer = Email(mailgun_api_key, sandbox_url, sender_email, recipient_email)
     mailer.send_email("Email Subject", "Email Body")
-    
-    
+
+## Testing
+
+### Unit testing
+
+All the unit tests are contained in the **/tests** folder. Run this tests when you want to check individual functions of the library.
```

Comment:
@vimalsheoran Please run these lines through Grammarly. I believe there are multiple errors.

Reply:
Yes, they might have gone unnoticed, will be fixed in the next commit.